courseaffils/templatetags/coursetags.py

```diff
@@ -95,6 +95,19 @@ def execute_query(self, user):
 register.tag('get_courses', GetCourses.process_tag)
 
 
+class GetInstructorCourses(TemplateTagNode):
+    noun_for = {"for": "user"}
+
+    def __init__(self, varname, user):
+        TemplateTagNode.__init__(self, varname, user=user)
+
+    def execute_query(self, user):
+        from courseaffils.views import get_courses_for_instructor
+        return get_courses_for_instructor(user)
+
+register.tag('get_instructor_courses', GetInstructorCourses.process_tag)
```

Comment:
I'm pretty sure we can simplify this a lot, and just do something like this:

```py
@register.simple_tag
def get_instructor_courses(user):
    from courseaffils.views import get_courses_for_instructor
    return get_courses_for_instructor(user)
```

per the instructions here: https://docs.djangoproject.com/en/1.11/howto/custom-template-tags/#writing-custom-template-tags

It looks like `TemplateTagNode` is doing some sophisticated parsing that isn't necessary if we use `simple_tag`